x-pack/plugins/siem/public/components/page/network/network_top_n_flow_table/columns.tsx

```diff
@@ -217,27 +215,27 @@ export const getNetworkTopNFlowColumns = (
 
 // Xavier fix this stuff under DO NOT FORGET
 
-const getIpTitle = (topNFlowType: NetworkTopNFlowType) => {
-  if (topNFlowType === NetworkTopNFlowType.source) {
+const getIpTitle = (flowTarget: FlowTarget) => {
```

Comment:
Is the `Xavier fix this stuff under DO NOT FORGET` above still needed?

Otherwise, I really do not know as a reader of the code what to not forget?

Reply:
[I was curious about this as well](https://github.com/elastic/kibana/pull/34208/files#r271033567) -- will touch base with @XavierM to determine for sure though.